### PR TITLE
updating rox-node, rox-browser, rox-react-native to Rox sdk 5

### PR DESCRIPTION
**Pull request by AsafRollout**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/rox-browser/index.d.ts [hunk 1]
```diff
@@ -1,60 +1,65 @@
-// Type definitions for rox-browser 4.8
+// Type definitions for rox-browser 5.0
 // Project: https://rollout.io
 // Definitions by: g-guirado <https://github.com/g-guirado>
+//                 rollout: <https://github.com/rollout>
```
Comment (sheetalkamat) [inline types/rox-browser/index.d.ts:4]: this needs to user name not org

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/rox-browser/index.d.ts (hunk 1 above)
Comment (AsafRollout) [reply types/rox-browser/index.d.ts:4]: sorry about that, thought i saw some companies github account.

can we use the company link (not github), like there are in the 'react' definition for example?
or is it only github users?

Comment (sheetalkamat) [reply types/rox-browser/index.d.ts:4]: These are suppose to be authors on GitHub maintaining the types so we can ping them for reviewing changes .. so yeah GitHub user id is needed here

**Review by rbuckton — APPROVED** [review]

**Comment by AsafRollout** [comment]
Ready to merge

**Pull request merged**